browser/eb_application.py

```diff
@@ -265,6 +268,35 @@ def create_connection(_):
         print(err)
     return conn
 
+def initial_connection(_):
+    ''' Create a connection to a MySQL server. Creates database if
+    it doesn't exist yet. Only called when starting app.
+    '''
+    try:
+        # connect but don't specify database: it might not exist
+        conn = MySQLdb.connect(
+            user=config.MYSQL_USERNAME,
+            host=config.MYSQL_HOSTNAME,
+            port=config.MYSQL_PORT,
+            passwd=config.MYSQL_PASSWORD,
+            charset='utf8',
+            use_unicode=True)
+
+        # on new server, caliban database won't exist yet
+        try:
+            query = '''CREATE DATABASE {}'''.format(config.MYSQL_DATABASE)
+            conn.cursor().execute(query)
+            conn.commit()
+
+        # it already exists
+        except:
+            pass
+
+        conn.close()
```

Comment:
Should put in [a `finally` block](https://docs.python.org/3/tutorial/errors.html#defining-clean-up-actions) in case there is an uncaught error in the try/except (assuming you change the error handling)